pkg/services/auth/authimpl/token_cleanup.go

```diff
@@ -66,3 +72,29 @@ func (s *UserAuthTokenService) deleteExpiredTokens(ctx context.Context, maxInact
 
 	return affected, err
 }
+
+func (s *UserAuthTokenService) deleteOrphanedExternalSessions(ctx context.Context) error {
```

Comment:
Should we add this to the store instead?

Reply:
I would leave it her to be next to the session token cleanup logic.